src/main/java/io/strimzi/kafka/bridge/mqtt/utils/MappingRule.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.kafka.bridge.mqtt.utils;
+
+import java.util.HashMap;
+
+/**
+ * Represents a Mapping Rule in the Topic Mapping Rules(TOMAR).
+ *
+ */
+public class MappingRule {
+    private String kafkaTopicTemplate;
+    private String mqttTopicPattern;
+    private HashMap<String, String> placeholdersType = new HashMap<>();
+
+    /**
+     * Constructor
+     *
+     * Creates a new instance of MappingRule.
+     * @param kafkaTopicTemplate the desired Kafka topic template. The kafka topic template can contain placeholders.
+     * @param mqttTopicPattern the mqtt topic pattern. The mqtt topic pattern can contain placeholders.
+     */
+    public MappingRule (String kafkaTopicTemplate, String mqttTopicPattern, HashMap<String, String> placeholdersType){
```

Comment:
What is the placeholdersType parameter? (Don't forget to add the @param Javadoc here too!)

Reply:
it would hold data type for each placeholder, e.g device/{id}/data, we should be able to define which data type the {id} placeholder will be of. 
but I am removing it as we do not need that now.